src/processor/operator/update/set_executor.cpp

```diff
@@ -1,14 +1,31 @@
 #include "processor/operator/update/set_executor.h"
 
+using namespace kuzu::common;
+
 namespace kuzu {
 namespace processor {
 
 void NodeSetExecutor::init(ResultSet* resultSet, ExecutionContext* context) {
     nodeIDVector = resultSet->getValueVector(nodeIDPos).get();
```

Comment:
Can we initialize `propertyVector` to nullptr in NodeSetExecutor's constructor?
Here we only need the `if` statement to set it to a valid value vector.